### DISCUSSION TITLE
Turkish Language Add

### DISCUSSION THREAD

**Discussion by atlantatr** [DirectoryLister/DirectoryLister#734] [Help & Support]
How can i add Turkish language to system?

**Comment by PHLAK** [comment]
You can add a file to the `app/translations` directory with contents following the pattern of the other files there (e.g. [`en.yaml`](https://github.com/DirectoryLister/DirectoryLister/blob/master/app/translations/en.yaml)).

If you do add a Turkish translation we'd happily accept a [pull request](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) to add this.

**Comment by atlantatr** [comment]
I already added and changed on config but after that change website doesn't works. If you can show me how to i will be appreciate. Here is the translation file https://easyupload.io/4v03ib

**Reply by PHLAK** [reply]
You also need to clear the cache by deleting the contents of the `app/cache` folder.

**Reply by atlantatr** [reply]
Now works perfect thanks

**Reply by PHLAK** [reply]
Would you be willing to share the translation file here? If so I can add it to the app so others can benefit as well.

**Reply by atlantatr** [reply]
Of course here Turkish translation you can add.
https://easyupload.io/daux9y

**Reply by PHLAK** [reply]
Thanks, [added](https://github.com/DirectoryLister/DirectoryLister/commit/878b969ea9a7bcfc2c0b317835f771e257fd180b)!
